content/nginx/deployment-guides/amazon-web-services/ingress-controller-elastic-kubernetes-services.md

```diff
@@ -76,90 +82,102 @@ In this guide we’re using the `eksctl` command to create an Amazon EKS cluster
 
    ```shell
-   kubectl apply -f service/loadbalancer-aws-elb.yaml
+   kubectl apply -f deployments/service/loadbalancer-aws-elb.yaml
    ```
 
 ### Enabling the PROXY Protocol
 
-1. Add the following keys to the <span style="white-space: nowrap; font-weight:bold;">common/nginx-config.yaml</span> config map file:
+1. Add the following keys to the <span style="white-space: nowrap; font-weight:bold;">deployments/common/nginx-config.yaml</span> config map file:
 
    ```yaml
    proxy-protocol: "True"
    real-ip-header: "proxy_protocol"
    set-real-ip-from: "0.0.0.0/0"
    ```
 
-2. Run the following command to update the config map:
+2. Run the following command to update the config map. This will enable the PROXY protocol for the NGINX Plus Ingress Controller:
 
    ```shell
-   kubectl apply -f common/nginx-config.yaml
+   kubectl apply -f deployments/common/nginx-config.yaml
    ```
 
-3. Enable the PROXY Protocol on the target group associated with the NLB created for your `LoadBalancer` service, by performing the steps in the **Enable Proxy Protocol** section of the [AWS documentation](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/load-balancer-target-groups.html#proxy-protocol).
+3. Enable the PROXY Protocol for the target groups linked to the NLB. Follow the steps in the **Enable proxy protocol** section of the [AWS documentation](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/edit-target-group-attributes.html#proxy-protocol) to do this.
 
 
 <span id="testing"></span>
 ## Testing
 
-1. Follow the [instructions](https://github.com/nginxinc/kubernetes-ingress/tree/master/examples/complete-example) for setting up our sample deployment of a demo app load balanced by the NGINX Plus Ingress controller.
-
-2. Navigate in a browser to **<http://cafe.example.com/coffee>** (or issue the `curl` command against that URL).
-
-3. Run the following command to display the name of the running Ingress pod:
+1. Get the DNS name of the NGINX Plus Ingress Controller with the command below. Find the value of the "LoadBalancer Ingress" in the output.
 
    ```shell
-   kubectl get pods -namespace=nginx-ingress
+   kubectl describe svc nginx-ingress --namespace=nginx-ingress
    ```
 
-4. Run the following command to display the logs from the NGINX Plus Ingress Controller, where `<pod_name>` is the name you learned in the previous step. If the logged IP address matches the IP address from which you accessed **<http://cafe> .example.com/coffee**, the PROXY Protocol is enabled.
+2. Resolve the DNS name into an IP address using `nslookup`:
 
    ```shell
-   kubectl logs nginx-ingress-<pod_name> -n nginx-ingress
+   nslookup <dns-name>
    ```
 
+3. Follow the [instructions](https://github.com/nginx/kubernetes-ingress/tree/main/examples/ingress-resources/complete-example) to deploy our Cafe demo app into the EKS cluster. It will be load balanced by your NGINX Plus Ingress Controller.
```

Comment:
```suggestion
3. Follow the [instructions](https://github.com/nginx/kubernetes-ingress/tree/main/examples/ingress-resources/complete-example) to deploy the Cafe demo app into the EKS cluster. It will be load balanced by NGINX Ingress Controller.
```